molecule/user_provided/prepare.yml

```diff
@@ -60,3 +70,52 @@
         GRANT ALL ON trillian.* to 'mysql'@'%' IDENTIFIED BY 'password' WITH GRANT OPTION;
         FLUSH PRIVILEGES;
         EOF
+
+    - name: Create a non-root sudoer user
```

Comment:
Why not create a file that does the user setup and then include it in the prepare files for both of the scenarios?

Reply:
Will do, never good to duplicate code